packages/cheerio-crawler/src/internals/cheerio-crawler.ts

```diff
@@ -25,10 +25,11 @@ export type CheerioErrorHandler<
 > = ErrorHandler<CheerioCrawlingContext<UserData, JSONData>>;
 
 export interface CheerioCrawlerOptions<
-    ExtendedContext extends CheerioCrawlingContext,
+    ContextExtension = {},
```

Comment:
do you really want to use `{}` as a type? it doesnt represent an empty object like in JS, for that you would need `Record` or similar. you can assign scalars to `{}` too.

Reply:
Yeah, I keep forgetting this. What would be the correct way to say "an empty object" then? `Record` has an index signature, and that's not what we want.

Reply:
`Record<PropertyKey, never>` i guess, or use `Dictionary<never>` if we only want to consider string keys.